### ISSUE TITLE
range-v3 fails to build: wrong archive

### ISSUE THREAD

**Issue by Fleischner** [Microsoft/vcpkg#1246]
Install of range-v3 fails with wrong hash.
      Expected hash: [ 615a53ebac22df03e865bdcc86580914c4505ec5fc691b6a2f864f7bf63690b99d0da0db2d5b1026e34b0a3f7557f30dfa0cad65643bae0b53c0ec066af9eb8e ]
        Actual hash: [ 6158cd9ee1f5957294a26dc780c881839e0bae8610688a618cd11d47df34d5e543fa09ac9a3b33d4a65af8eceae0a6a3055621206c291ef75f982e7915daf91a ]

I realized, that the version supplied with vcpkg currently is quite old (probably for compatibility reasons with VS2015).
Is there a plan to upgrade to a newer version, or is this blocked by backwards compatibility?
The current tagged version on github is 0.2.5. Has anyone tried? Does this work with VS2017 out of the box, or are patches necessary?

**Comment by ras0219-msft** [comment]
Thanks for reporting this issue!

My current understanding is that the upstream version on GitHub still does not work with MSVC out of the box.

Also, I believe we have already fixed that hash issue -- the latest port version is now `20151130-vcpkg4`. Could you try again with the latest portfiles and let us know if you still have issues? The current expected hash should be
> a1dad795b15c8491963f75fb58097dd290203406038cb2790c66b7ebd854a42043a92cfa2fed30c125529c3aa8d0cfc05cf1d149c84e96eb1f865857ad87adeb

**Comment by Fleischner** [comment]
You are right. Works. Sorry my fault. Made an error during upgrade to the newest version.

**Comment by JackBoosY** [comment]
Please open a new issue if this is still a problem for you.
Thx.
